torch/_inductor/codegen/cuda/gemm_template.py

```diff
@@ -229,19 +254,70 @@ def has_tma_epilogue(op) -> bool:
         return result
 
     @staticmethod
+    def supports_evt(op: "cutlass_gemm_op.GemmOperation") -> bool:  # type: ignore[name-defined]
```

Comment:
Helper method to determine whether a cutlass GemmOperation supports EVT Fusion or not.